static/js/apps/eval_rig/data_store.tsx

```diff
@@ -37,13 +38,28 @@ import {
 } from "./constants";
 import { Response } from "./feedback_form";
 
+// Add a new doc for a call
+async function addCallToStore(
+  sheetId: string,
+  queryId: string,
+  callId: string
+): Promise<void> {
+  const docRef = doc(
+    collection(db, "sheets", sheetId, "queries", queryId, "calls"),
+    callId
+  );
+  // Need to set fields in a document in order to create the document
+  setDoc(docRef, { dummy: "dummy" });
```

Comment:
nit: maybe set it {id: callId} ?

Reply:
If there is already a call entry with response, when we allow multiple evals and another response is added, would this still be okay?

i am not sure if setDoc is a pure reset (we don't want that), or just adding a new field (desired)

Reply:
done

Reply:
reading through documentation, sub-collection and document are relatively independent, so this is ok!